### ISSUE TITLE
Issue with the booleans in graphs

### ISSUE THREAD

**Issue by epigramx** [freqtrade/frequi#485]
They seem to not work properly currently **_on the tooltip_**. They show a "-" always; they used to be False or True; they appear on the graph itself properly.

 - OS: Windows
 - Browser: Chrome
 - Version: latest stable

**Comment by xmatthias** [comment]
it's unclear what you mean with this.

maybe you're able to provide a screenshot - or better describe what you mean with "booleans in graph".

**Comment by epigramx** [comment]
> it's unclear what you mean with this.
> 
> maybe you're able to provide a screenshot - or better describe what you mean with "booleans in graph".
> If it's related to data from a strategy, please provide a strategy so it can be easily recreated.

It's just if you hover over the graph itself, you can a big tooltip that has all the variables used in the graph itself. The "lines" that were based on booleans and not floats used to be denoted with "False" or "True" and now they have a dash ("-") (they are drawn properly though).

**Comment by xmatthias** [comment]
using *bold* text will not help me reproduce it.

Please provide a simple strategy where this is happening (ideally with screenshots) so it can be easily recreated.

Otherwise this will probably remain an issue that might remain open for a long time, as it requires people trying to fix it to first create a strategy where this is might be happening (with the need to rely on your description of something happening - which means it's potentially not clear what to look out for).

**Comment by epigramx** [comment]
Are you saying you hover over the tooltip, and see "True" or "False" on booleans used in the graph? I don't want to reply further if even the concept of the problem itself is not understood.

**Comment by epigramx** [comment]
I don't think we shoul turn into art designers to explain concepts like this, because I think language was enough, but here you go 
![image](https://user-images.githubusercontent.com/2960257/132873703-91c27712-f086-45fc-80af-6d06330c9383.png)

**Comment by epigramx** [comment]
Hovering over a graph with the mouse cursor you get a tooltip like this: 
![image](https://user-images.githubusercontent.com/2960257/132874742-c8cbe3d6-8c7c-4dca-8e64-a3fc5b140e20.png)

You can see 'open' is a float and you see it on the right.

The problem is that booleans like 'Buy' are dashed.

**Comment by Bloodhunter4rc** [comment]
yes having the same "issue" this should display recentispumping = False and not "-"
![s1](https://user-images.githubusercontent.com/8630485/135090420-3fcc3ee7-62ac-4c3a-8d9b-745232245552.PNG)

used in freqtrade like this: 
dataframe['recentispumping'] = (dataframe['ispumping'].rolling(300).max() > 0) | (dataframe['islongpumping'].rolling(300).max() > 0) | (dataframe['isvolumepumping'].rolling(300).max() > 0)
which returns True or False

**Comment by xmatthias** [comment]
i'm not convinced this is something that a change to freqUI caused - but more likely a change in echarts, which now reacts differently to boolean values when displaying them in tooltips (if this ever worked, it was probably by accident).

as such - the best workaround for this is to explicitly assign 0 / 1 in the strategy - for example using 

``` python
dataframe['recentispumping'] = np.where((dataframe['ispumping'].rolling(300).max() > 0) | (dataframe['islongpumping'].rolling(300).max() > 0) | (dataframe['isvolumepumping'].rolling(300).max() > 0), 1, 0)
```

**Comment by xmatthias** [comment]
Fixed with #571.
